pkg/utils/kubeconfig/kubeconfig.go

```diff
@@ -180,6 +203,30 @@ func AppendAuthenticator(config *clientcmdapi.Config, clusterMeta *api.ClusterMe
 	}
 }
 
+// AWSAuthenticatorVersionFormat is the format in which aws-iam-authenticator displays version information:
+// {"Version":"0.5.5","Commit":"85e50980d9d916ae95882176c18f14ae145f916f"}
+type AWSAuthenticatorVersionFormat struct {
+	Version string `json:"Version"`
+}
+
+func authenticatorIsAboveVersion(version string) (bool, error) {
+	cmd := exec.Command(AWSIAMAuthenticator, "version")
+	output, err := cmd.Output()
+	if err != nil {
+		return false, fmt.Errorf("failed to run aws-iam-authenticator version command: %w", err)
+	}
+	var parsedVersion AWSAuthenticatorVersionFormat
+	if err := json.Unmarshal(output, &parsedVersion); err != nil {
+		return false, fmt.Errorf("failed to parse version information: %w", err)
+	}
+
+	compareVersions, err := utils.CompareVersions(parsedVersion.Version, version)
+	if err != nil {
+		return false, fmt.Errorf("failed to parse versions: %w", err)
+	}
+	return compareVersions > 1 || compareVersions == 0, nil
```

Comment:
```suggestion
	return compareVersions >= 1 || compareVersions == 0, nil
```

doesn't it return `1`, not a value greater than `1`?

Reply:
```
// returns 0 if a == b
// returns 1 if a > b
// returns -1 if a < b
```